packages/dialog/index.tsx

```diff
@@ -258,7 +260,7 @@ class Dialog<T extends {} = HTMLElement> extends React.Component<
     const container: React.ReactElement<HTMLDivElement> | undefined =
       this.renderContainer(children);
     return (
-      // @ts-ignore
+      // @ts-ignore Tag does not have any construct
```

Comment:
is there a github issue associated with this?

Reply:
Yes, an issue you opened ;)

https://github.com/Microsoft/TypeScript/issues/28892

Perhaps I phrased it wrong

Reply:
sorry - I poorly communicated. Could you please add that issue to the comment on this line